action_log/.idea/workspace.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A lot of these files are IntelliJ/AS specific. 
Can you add them to .gitignore if appropriate. 
This one should be ignored https://stackoverflow.com/questions/27647889/should-i-add-idea-workspace-xml-to-gitignore